showcase-app/main.js

```diff
@@ -1,69 +1,80 @@
-import {
-  fetch,
-  login,
-  getDefaultSession,
-  handleIncomingRedirect,
-} from '@inrupt/solid-client-authn-browser';
-import {
-  getSolidDataset,
-  getThing,
-  getStringNoLocale,
-} from '@inrupt/solid-client';
-import { SCHEMA_INRUPT } from '@inrupt/vocab-common-rdf';
+import {fetch, getDefaultSession, handleIncomingRedirect, login,} from '@inrupt/solid-client-authn-browser';
+import {getSolidDataset, getStringNoLocale, getThing,} from '@inrupt/solid-client';
+import {SCHEMA_INRUPT} from '@inrupt/vocab-common-rdf';
 import IdentityWidget from './plugin/identityPlugin';
```

Comment:
Rename `IdentityWidget`. Choose one word: either plugin, extension or widget. Don't use three different words. This leads to confusion.

Reply:
✔️